docs/src/pages/components/pagination/examples/PaginationHasMorePagesExample.tsx

```diff
@@ -0,0 +1,64 @@
+import { useState } from 'react';
```

Comment:
```suggestion
import * as React from 'react';
```
https://epicreact.dev/importing-react-through-the-ages/